docs/src/main/asciidoc/native-and-ssl-guide.adoc

```diff
@@ -178,29 +181,16 @@ Now, the application should work as expected:
  * you should see a JSON output with some information about Greece
  * hit `Ctrl+C` to stop the application
 
-[TIP]
-====
-The root certificates file of GraalVM might not be totally up to date.
-If you have issues with some certificates, your best bet is to include the `cacerts` file of a regular JDK instead.
-====
-
-[WARNING]
-====
-Don't forget to move your GraalVM directory back to where it was.
-====
-
-=== Working with containers
-
```

Comment:
I don't think we want these changes, do we?

Reply:
Ooops, my IDE was not synced with recent changes. I'll revert this.